src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Dropdown from './Dropdown';
+
+export default {
+  title: 'Components/Dropdown',
+  component: Dropdown,
+} as ComponentMeta<typeof Dropdown>;
+
+const Template: ComponentStory<typeof Dropdown> = (args) => (
+  <Dropdown {...args} />
+);
+
+export const Default = Template.bind({});
```

Comment:
In Storybook, the state is not selected from the dropdown. I also see that we lose the border onClick. Is this just storybook? The control can be added in a another issue.

Reply:
I don't know if there is a way in Storybook to assign state in stories. And about the border, I think it changes to white, if you would check dropdown in topics for example. We have gray background there so it looks alright I think.

Reply:
I also ran into this problem with state in stories.
If you make handleChange optional in Dropdown.tsx and then don't assign it in Dropdown.stories.tsx then it handles the stat automatically and it works somehow.
```
// Drowpdown.stories.tsx

export const Default = Template.bind({});
Default.args = {
  values: ['days', 'months', 'years'],
};
```
I'll make an issue to look into that, there must be a way to control the state manually.

The border disappears when the dropdown is in focus, can you set the focus color in the MUI template?

Reply:
@DomyS , once you have added please resolve conversation yourself.